index.bs

```diff
@@ -1043,6 +1067,7 @@ dictionary WebTransportHash {
 
 dictionary WebTransportOptions {
   boolean allowPooling;
+  boolean disallowFallback;
```

Comment:
From @martinthomson 
```suggestion
  boolean requireUnreliable;
```

Reply:
Meeting:
- Depends on whether we want fallback to be the default or not.
- Early adopters can specify `requireUnreliable: true` now to avoid fallback in the future.
- Should get this in early so people won't be surprised by this behavior.
- Different from allowPooling... open new issue to discuss allowPooling default?